### DISCUSSION TITLE
WingetUI don't work on my Win 11 ?

### DISCUSSION THREAD

**Discussion by sypqys** [Devolutions/UniGetUI#1668] [Q&A] [answered]
Hello !

I don't understand. There is NoVirusThanks SysHardener which is the source of this problem ? Maybe because I have launch this...
Or DefenderUI... I don't know...
![ScreenWings_tJUgsIhviu](https://github.com/marticliment/WingetUI/assets/27967268/0e88339a-98f3-443a-8e86-37a846234fee)
![ScreenWings_Gn6oQwiqGf](https://github.com/marticliment/WingetUI/assets/27967268/fa9e4749-52c3-4222-a134-8778caaf5908)

it is installed, but the logo appear and disappear and nothing happens, nothing on taskbar manager process...
What I have to do ?
Which kind of script are exectuted with WingetUI. Because on Hard_Configurator, I have put on "block sponsors" also...
But I thing, the problem was caused by SysHardener...

Thanks

**Comment by marticliment** [comment]
Hello,

Which WingetUI version are you attempting to install? From where are you attempting to install it?
Disabling all AV software allowa you to install WingetUI successfully?

**Comment by sypqys** [comment]
2.0.3 my version.

**Reply by marticliment** [reply]
Version 2.0.3 is quite old, please test version 2.2.0 (download from here: https://github.com/marticliment/WingetUI/releases/download/2.2.0/WingetUI.Installer.exe)

**Comment by sypqys** [comment]
![ScreenWings_eG1HSfpQrd](https://github.com/marticliment/WingetUI/assets/27967268/32b175db-f8a8-412e-82ed-08cf1cbb07f3)

**Reply by sypqys** [reply]
It is updated, but the problem, it don't work.. see screen capture...

**Reply by marticliment** [reply]
Can you please share your Logs? (Both WingetUI Logs and Package Manager Logs)

**Reply by sypqys** [reply]
I don't know where they are (the logs) ?

**Reply by sypqys** [reply]
[wingetUI journaux.txt](https://github.com/marticliment/WingetUI/files/13798325/wingetUI.journaux.txt)

**Reply by marticliment** [reply, marked as answer]
As seen on your log, winget.exe is being blocked by a Group Policy. Are you trying to install WingetUI on a managed computer? (your work laptop/pc)
![image](https://github.com/marticliment/WingetUI/assets/53119851/0fde0dcc-b875-4a15-874c-114d5cbcef1e)

**Reply by sypqys** [reply]
![ScreenWings_dGpMuH8mFy](https://github.com/marticliment/WingetUI/assets/27967268/1b0c409a-f539-4c99-9116-1f8d7c4af138)
